exercises/concept/karls-languages/.meta/design.md

```diff
@@ -43,6 +43,19 @@ This Concepts Exercise's Concepts are:
 
 This Concept Exercise's prerequisites Concepts are:
 
-- `for-loops`: know how to use a for-loop to iterate over a collection.
 - `arrays`: know of the array collection type and that it has a fixed length.
 - `strings`: data types used in this exercise
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer]:
+
+- `essential`: If the student did not reuse the implementation of the `containsLanguage` method in the `isExciting` method, instruct them to do so.
+  Explain that reusing existing code instead of copy-pasting can help make code easier to maintain.
+- `actionable`: If the solution did not use `contains` in the method `containsLanguage`, instruct the student to do so.
+- `actionable`: If the solution did not use `isEmpty` in the method `isEmpty`, instruct the student to do so.
+
```

Comment:
Based on some of the solution sets I found in the community solutions we may want to add some (`informative`?) feedback on using

```java
return languages.contains(language);
```

instead of

```java
if (languages.contains(language) {
    return true;
}
return false;
```

Reply:
Done!